### DISCUSSION TITLE
运行检测模型时候有的样本数据报错：IndexError: list index out of range

### DISCUSSION THREAD

**Discussion by XD-mu** [PaddlePaddle/PaddleOCR#12739] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
在训练检测模型时候检测有的数据的时候会报这个错误，我当时用的是paddleLabel做的数据集，有的一张照片有几十个样本，不知道在哪里可以设置这个阈值让它不报错并且进行训练，麻烦大家了
- 系统环境/System Environment：Win10
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：PaddleOCR-release-2.6
- 运行指令/Command Code：python tools/train.py -c xxxxxx/xxxxx/xxxxx/xxxx.yml
- 完整报错/Complete Error Message：[2022/12/30 18:42:24] ppocr ERROR: When parsing line 密集文本数据集/22.png      [{"transcription": "3rLl", "points": [[410.0, 30.0], [609.0, 30.0], [609.0, 49.0], [410.0, 49.0]], "difficult": false}, {"transcription": "NI", "points": [[137.0, 42.0], [154.0, 42.0], [154.0, 51.0], [137.0, 51.0]], "difficult": false}, {"transcription": "654k", "points": [[308.0, 80.0], [376.0, 80.0], [376.0, 88.0], [308.0, 88.0]], "difficult": false}, {"transcription": "FFFPFFPPPPPPPPP", "points": [[130.0, 101.0], [383.0, 100.0], [383.0, 115.0], [130.0, 116.0]], "difficult": false}, {"transcription": "CJzX", "points": [[136.0, 136.0], [196.0, 136.0], [196.0, 145.0], [136.0, 145.0]], "difficult": false}, {"transcription": "Eg*", "points": [[256.0, 130.0], [284.0, 130.0], [284.0, 137.0], [256.0, 137.0]], "difficult": false}, {"transcription": "JucD", "points": [[415.0, 128.0], [483.0, 126.0], [483.0, 137.0], [415.0, 139.0]], "difficult": false}, {"transcription": "lj_", "points": [[88.0, 143.0], [126.0, 143.0], [126.0, 152.0], [88.0, 152.0]], "difficult": false}, {"transcription": "B", "points": [[132.0, 162.0], [160.0, 163.0], [160.0, 170.0], [132.0, 169.0]], "difficult": false}, {"transcription": "Xw", "points": [[213.0, 158.0], [235.0, 160.0], [233.0, 174.0], [212.0, 172.0]], "difficult": false}, {"transcription": "X3", "points": [[388.0, 162.0], [414.0, 162.0], [414.0, 174.0], [388.0, 174.0]], "difficult": false}, {"transcription": "BuFl4", "points": [[26.0, 181.0], [118.0, 184.0], [117.0, 200.0], [25.0, 197.0]], "difficult": false}, {"transcription": "1FI", "points": [[30.0, 172.0], [129.0, 173.0], [129.0, 184.0], [30.0, 183.0]], "difficult": false}, {"transcription": "Lrx", "points": [[120.0, 187.0], [161.0, 184.0], [162.0, 197.0], [121.0, 200.0]], "difficult": false}, {"transcription": "8K", "points": [[388.0, 178.0], [412.0, 178.0], [412.0, 185.0], [388.0, 185.0]], "difficult": false}, {"transcription": "XDS", "points": [[386.0, 189.0], [415.0, 189.0], [415.0, 201.0], [386.0, 201.0]], "difficult": false}, {"transcription": "DFX6I", "points": [[26.0, 211.0], [132.0, 211.0], [132.0, 224.0], [26.0, 224.0]], "difficult": false}, {"transcription": "XD6", "points": [[388.0, 205.0], [414.0, 205.0], [414.0, 214.0], [388.0, 214.0]], "difficult": false}, {"transcription": "md", "points": [[133.0, 215.0], [160.0, 215.0], [160.0, 223.0], [133.0, 223.0]], "difficult": false}, {"transcription": "G", "points": [[391.0, 214.0], [412.0, 215.0], [411.0, 227.0], [390.0, 226.0]], "difficult": false}, {"transcription": "A", "points": [[552.0, 207.0], [562.0, 207.0], [562.0, 216.0], [552.0, 216.0]], "difficult": false}, {"transcription": "BKm", "points": [[381.0, 232.0], [412.0, 232.0], [412.0, 239.0], [381.0, 239.0]], "difficult": false}, {"transcription": "1I3krI5", "points": [[138.0, 245.0], [184.0, 245.0], [184.0, 250.0], [138.0, 250.0]], "difficult": false}, {"transcription": "CL", "points": [[374.0, 244.0], [410.0, 244.0], [410.0, 253.0], [374.0, 253.0]], "difficult": false}, {"transcription": "lI", "points": [[609.0, 251.0], [622.0, 251.0], [622.0, 259.0], [609.0, 259.0]], "difficult": false}, {"transcription": "3L", "points": [[3.0, 266.0], [30.0, 266.0], [30.0, 273.0], [3.0, 273.0]], "difficult": false}, {"transcription": "GI", "points": [[388.0, 261.0], [406.0, 261.0], [406.0, 265.0], [388.0, 265.0]], "difficult": false}, {"transcription": "3MD", "points": [[136.0, 271.0], [159.0, 271.0], [159.0, 277.0], [136.0, 277.0]], "difficult": false}, {"transcription": "_II", "points": [[430.0, 269.0], [471.0, 269.0], [471.0, 274.0], [430.0, 274.0]], "difficult": false}, {"transcription": "BWU", "points": [[41.0, 282.0], [120.0, 282.0], [120.0, 290.0], [41.0, 290.0]], "difficult": false}, {"transcription": "vDDI", "points": [[385.0, 283.0], [418.0, 283.0], [418.0, 296.0], [385.0, 296.0]], "difficult": false}, {"transcription": "T3HD", "points": [[135.0, 299.0], [167.0, 299.0], [167.0, 304.0], [135.0, 304.0]], "difficult": false}, {"transcription": "RXB", "points": [[379.0, 294.0], [416.0, 294.0], [416.0, 309.0], [379.0, 309.0]], "difficult": false}, {"transcription": "Cm", "points": [[27.0, 304.0], [47.0, 305.0], [46.0, 315.0], [26.0, 313.0]], "difficult": false}, {"transcription": "13Lm", "points": [[27.0, 318.0], [164.0, 321.0], [163.0, 334.0], [27.0, 331.0]], "difficult": false}, {"transcription": "x", "points": [[382.0, 312.0], [412.0, 312.0], [412.0, 322.0], [382.0, 322.0]], "difficult": false}, {"transcription": "8x3", "points": [[383.0, 319.0], [414.0, 327.0], [411.0, 341.0], [379.0, 333.0]], "difficult": false}, {"transcription": "D", "points": [[26.0, 331.0], [122.0, 331.0], [122.0, 345.0], [26.0, 345.0]], "difficult": false}, {"transcription": "BFK", "points": [[24.0, 341.0], [77.0, 341.0], [77.0, 361.0], [24.0, 361.0]], "difficult": false}, {"transcription": "X3", "points": [[381.0, 339.0], [412.0, 339.0], [412.0, 349.0], [381.0, 349.0]], "difficult": false}, {"transcription": "-", "points": [[598.0, 335.0], [620.0, 335.0], [620.0, 342.0], [598.0, 342.0]], "difficult": false}, {"transcription": "8B", "points": [[24.0, 351.0], [58.0, 353.0], [56.0, 376.0], [22.0, 374.0]], "difficult": false}, {"transcription": "x", "points": [[139.0, 349.0], [167.0, 349.0], [167.0, 359.0], [139.0, 359.0]], "difficult": false}, {"transcription": "G2Bzuu", "points": [[134.0, 362.0], [217.0, 367.0], [216.0, 382.0], [133.0, 377.0]], "difficult": false}, {"transcription": "C", "points": [[390.0, 353.0], [405.0, 353.0], [405.0, 361.0], [390.0, 361.0]], "difficult": false}, {"transcription": "Lf_", "points": [[104.0, 363.0], [130.0, 363.0], [130.0, 368.0], [104.0, 368.0]], "difficult": false}, {"transcription": "m", "points": [[305.0, 369.0], [334.0, 369.0], [334.0, 379.0], [305.0, 379.0]], "difficult": false}, {"transcription": "-", "points": [[605.0, 372.0], [621.0, 372.0], [621.0, 382.0], [605.0, 382.0]], "difficult": false}, {"transcription": "NkMmHgr", "points": [[226.0, 394.0], [310.0, 392.0], [311.0, 404.0], [226.0, 407.0]], "difficult": false}, {"transcription": "II", "points": [[163.0, 449.0], [196.0, 449.0], [196.0, 462.0], [163.0, 462.0]], "difficult": false}, {"transcription": "BT", "points": [[210.0, 453.0], [270.0, 453.0], [270.0, 462.0], [210.0, 462.0]], "difficult": false}, {"transcription": "WjlbEFX", "points": [[468.0, 448.0], [566.0, 448.0], [566.0, 467.0], [468.0, 467.0]], "difficult": false}, {"transcription": "X_", "points": [[163.0, 467.0], [183.0, 467.0], [183.0, 481.0], [163.0, 481.0]], "difficult": false}, {"transcription": "I", "points": [[301.0, 472.0], [321.0, 474.0], [320.0, 480.0], [301.0, 478.0]], "difficult": false}, {"transcription": "NW", "points": [[163.0, 489.0], [189.0, 489.0], [189.0, 504.0], [163.0, 504.0]], "difficult": false}, {"transcription": "L", "points": [[410.0, 493.0], [503.0, 497.0], [503.0, 512.0], [409.0, 508.0]], "difficult": false}, {"transcription": "jbE", "points": [[465.0, 487.0], [568.0, 487.0], [568.0, 510.0], [465.0, 510.0]], "difficult": false}, {"transcription": "mmmmm", "points": [[162.0, 504.0], [252.0, 504.0], [252.0, 521.0], [162.0, 521.0]], "difficult": false}, {"transcription": "LCG2B", "points": [[255.0, 532.0], [336.0, 534.0], [336.0, 552.0], [254.0, 550.0]], "difficult": false}]
, error happened with msg: Traceback (most recent call last):
  File "F:\OCR_Project\PaddleOCR-release-2.6\ppocr\data\simple_dataset.py", line 137, in __getitem__
    outs = transform(data, self.ops)
  File "F:\OCR_Project\PaddleOCR-release-2.6\ppocr\data\imaug\__init__.py", line 56, in transform
    data = op(data)
  File "F:\OCR_Project\PaddleOCR-release-2.6\ppocr\data\imaug\make_border_map.py", line 60, in __call__
    self.draw_border_map(text_polys[i], canvas, mask=mask)
  File "F:\OCR_Project\PaddleOCR-release-2.6\ppocr\data\imaug\make_border_map.py", line 81, in draw_border_map
    padded_polygon = np.array(padding.Execute(distance)[0])
IndexError: list index out of range

**Comment by XD-mu** [comment]
是配置文件的设置问题吗？

**Comment by zhangxp12** [comment]
> 是配置文件的设置问题吗？

您好，我也是这个问题，数据集从官方下载的，最后你咋弄的？

**Comment by lipeng1109** [comment]
你好，我也出现了这个问题，这个不解决会影响最终结果吗？

**Comment by xuxiansheng2018** [comment]
你好，我也出现了这个问题，应该如何解决？

**Comment by wangchenhong** [comment]
你好，我也遇到了一样的问题。请问解决了吗

**Comment by SWHL** [comment]
怀疑是样本数据框坐标有超出图像边界的，才会报这个错误
